cvts/models.py

```diff
@@ -13,11 +14,24 @@
 
 DBase = declarative_base()
 
+vehicle_type_association_table = Table('vehicle_vehicle_type_assoc', DBase.metadata,
+    Column('vehicle_id', Integer, ForeignKey('vehicles.id')),
+    Column('vehicle_type_id', Integer, ForeignKey('vehicle_types.id'))
+)
+
 class Vehicle(DBase):
     """Data on a vehicle."""
     __tablename__ = 'vehicles'
     id            = Column(Integer, primary_key=True)
-    rego          = Column(String)
+    vehicle_types = relationship("VehicleType", secondary=vehicle_type_association_table)
```

Comment:
I am probably missing something, but vehicle type is not unique

Reply:
See my mesage on Slack. Vehicle type may as well be unique if we disregard "unclassified"

Reply:
The table defined by *vehicle_type_association_table* (table *vehicle_vehicle_type_assoc* in the DB) contains the many2many.